OutSystems/outsystems-ui
# ROU-4649: Fix issue on arrow navigation when a TabHeader is disabled

This PR is for fix an issue on arrow navigation when the tabs header is disabled.

### What was happening
- When Tabs pattern contains disabled tabs header inside, it's possible to reach the element disabled using the arrows left/right
![IssueTabsDisable](https://github.com/OutSystems/outsystems-ui/assets/25321845/20c2e2d7-2492-4fef-ab92-b080ee632310)

### What was done
- Add a new list of enabled items
- Review the **_handleKeypressEvent** method to use the new list of enabled elements
- Review each key behavior to lock the navigation for disabled items
![2023-12-04 17 44 34](https://github.com/OutSystems/outsystems-ui/assets/25321845/5856342f-9991-44bd-932a-81c8a7e6b929)

### Test Steps
1. Open Sample Page
2. Navigate to tabs using Tab key
3. On tabs header, use the arrows to navigate between elements
4. Expected: The disabled tabs header can't be reached or selected.

### Screenshots

(prefer animated gif)

### Checklist
